weblate/machinery/base.py

```diff
@@ -82,6 +82,7 @@ def request(self, method, url, skip_auth=False, **kwargs):
         headers = {
             'Referer': get_site_url(),
             'Accept': 'application/json; charset=utf-8',
+            'Content-Type': 'application/json',
```

Comment:
I'm not sure all machineries use JSON, so this might be wrong. Also requests set this automatically when you pass `json` to request, so it's not needed in your case.

Reply:
Yes, this is a mistake, i would not say it!s good idea to touch base.py unless changes needs to be reflected to all machines. Any changes there are mistake.